client/src/components/Shared/AverageStars.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import helper from './helpers.js'
+import FullStar from './../Review Widget/icons/star.svg'
+import HalfStar from './../Review Widget/icons/star-half.svg'
+import QuarterStar from './../Review Widget/icons/star-one-quarter.svg'
+import ThreeQuarterStar from './../Review Widget/icons/star-three-quarter.svg'
+import EmptyStar from './../Review Widget/icons/empty-star.svg'
+
+const AverageStars = (props) => {
+  const averageRating = helper.findStarRating(props.rating);
+
+  let wholeRating = Math.floor(averageRating) || 0;
+
+  let wholeStars = Array.from(Array(wholeRating)).map((_, i) => (
```

Comment:
Using '_' as a placeholder in map which holds the value of the array at each position. The map is using the index which is the 2nd parameter and incrementing it by 1 to return 1-5 instead of 0-4 so it is not using the first parameter. Kinda of 'hacky'.